Sources/KeyboardSupport/UIView+KeyboardSupport.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UIView+KeyboardSupport.swift
+//  KeyboardSupport
+//
+//  Created by John Davis on 12/3/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    
+    /// highest level superview of the view hierarchy
+    var topLevelContainer: UIView {
+        var returnView = self
+        
+        while returnView.superview != nil {
+            returnView = returnView.superview!
```

Comment:
Instead of force unwrapping, let's use `while let` instead.
```swift
    var topLevelContainer: UIView {
        var returnView = self
        
        while let superview = returnView.superview {
            returnView = superview
        }
        
        return returnView
    }
```

Reply:
Neat!
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/a126c05d41e49da9d35df406cc9e2304895f5984